src/scenes/Leaderboard/index.js

```diff
@@ -0,0 +1,73 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import { injectIntl, defineMessages } from 'react-intl';
+import { Grid, withStyles, Card as _Card, CardContent, Typography } from '@material-ui/core';
+
+import styles from './styles';
+import _Leaderboard from '../Event/components/Leaderboard';
+import { Row, Content } from '../Event/components';
+
+const messages = defineMessages({
+  totalEvents: {
+    id: 'leaderboard.totalEvents',
+    defaultMessage: 'Total Events',
+  },
+  totalParticipants: {
+    id: 'leaderboard.totalParticipants',
+    defaultMessage: 'Total Participants',
+  },
+  totalQTUM: {
+    id: 'leaderboard.totalQTUM',
+    defaultMessage: 'Total bet in QTUM',
+  },
+  totalBOT: {
+    id: 'leaderboard.totalBOT',
+    defaultMessage: 'Total bet in BOT',
+  },
+});
+
+@withStyles(styles, { withTheme: true })
+@injectIntl
+@inject('store')
+@observer
+export default class Leaderboard extends Component {
+  componentDidMount() {
+    this.props.store.leaderboard.init();
+  }
+  render() {
+    const { eventCount, participantsCount, totalBOT, totalQTUM } = this.props.store.leaderboard;
+    const { intl } = this.props;
+    return (
+      <Row>
+        <SidebarContainer>
+          <Card title={intl.formatMessage(messages.totalEvents)} value={eventCount} />
```

Comment:
pass the id only to the title so you only have to call `intl.formatMessage()` once in `Card`.